pkg/util/docker/cgroup.go

```diff
@@ -446,8 +446,8 @@ func parseCgroupMountPoints(r io.Reader) map[string]string {
 	scanner := bufio.NewScanner(r)
 	for scanner.Scan() {
 		mount := scanner.Text()
-		if strings.HasPrefix(mount, "cgroup ") {
-			tokens := strings.Split(mount, " ")
+		tokens := strings.Split(mount, " ")
+		if len(tokens) >= 3 && tokens[2] == "cgroup" {
```

Comment:
As I had to dig into `/etc/mtab` to understand what tokens[2] means, could you please add a `checking filesystem type is cgroup` comment?